lib/openshift/networkattachmentdefinition.rb

```diff
@@ -0,0 +1,9 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # represnets an Openshift Namespace
+  class Networkattachmentdefinition < ClusterResource
+    RESOURCE = 'networkattachmentdefinition.k8s.cni.cncf.io'
```

Comment:
could you paste `-o yaml` output of such object? I think it might be `networkattachmentdefinitions.k8s.cni.cncf.io`

Reply:
apiVersion: k8s.cni.cncf.io/v1
kind: NetworkAttachmentDefinition
metadata:
  creationTimestamp: "2019-11-12T00:20:43Z"
  generation: 1
  name: macvlan-bridge
  namespace: test
  resourceVersion: "190189"
  selfLink: /apis/k8s.cni.cncf.io/v1/namespaces/test/network-attachment-definitions/macvlan-bridge
  uid: fd2946d2-3608-4fa1-b174-52cd944ea057
spec:
  config: '{ "cniVersion": "0.3.0", "type": "macvlan", "master": "eth0", "mode": "bridge",
    "ipam": { "type": "host-local", "subnet": "10.1.1.0/24", "rangeStart": "10.1.1.100",
    "rangeEnd": "10.1.1.200", "routes": [ { "dst": "0.0.0.0/0" } ], "gateway": "10.1.1.1"
    } }'

Reply:
nope..no 's'
$ oc get networkattachmentdefinitions.k8s.cni.cncf.io
error: the server doesn't have a resource type "networkattachmentdefinitions"

Reply:
The thing we are looking at here is the `selfLink` where we actually see the surprising `network-attachment-definitions`. I see dashes for the first time.